src/main/java/org/broadinstitute/hellbender/tools/exome/segmentation/PerformJointSegmentation.java

```diff
@@ -0,0 +1,126 @@
+    protected static final String INITIAL_NUM_COPY_RATIO_STATES_LONG_NAME = "initialNumberOfCopyRatioStates";
+    protected static final String INITIAL_NUM_COPY_RATIO_STATES_SHORT_NAME = "initialNumCRStates";
+
+    protected static final String INITIAL_NUM_ALLELE_FRACTION_STATES_LONG_NAME = "initialNumberOfAlleleFractionStates";
+    protected static final String INITIAL_NUM_ALLELE_FRACTION_STATES_SHORT_NAME = "initialNumAFStates";
+
+    @Argument(
+            doc = "Tangent-normalized log2 read counts file",
+            shortName = ExomeStandardArgumentDefinitions.TANGENT_NORMALIZED_COUNTS_FILE_SHORT_NAME,
+            fullName =  ExomeStandardArgumentDefinitions.TANGENT_NORMALIZED_COUNTS_FILE_LONG_NAME,
+            optional = false
+    )
+    protected String coverageFile;
+
+    @Argument(
+            doc = "Input file for tumor-sample ref/alt read counts at normal-sample heterozygous-SNP sites (output of GetHetCoverage tool).",
+            fullName = ExomeStandardArgumentDefinitions.TUMOR_ALLELIC_COUNTS_FILE_LONG_NAME,
+            shortName = ExomeStandardArgumentDefinitions.TUMOR_ALLELIC_COUNTS_FILE_SHORT_NAME,
+            optional = false
+    )
+    protected File snpCountsFile;
+
+    @Argument(
+            doc = "Initial number of hidden copy-ratio states",
+            fullName = INITIAL_NUM_COPY_RATIO_STATES_LONG_NAME,
+            shortName = INITIAL_NUM_COPY_RATIO_STATES_SHORT_NAME,
+            optional = false
+    )
+    protected int initialNumCRStates;
+
+    @Argument(
+            doc = "Initial number of hidden allele-fraction states",
+            fullName = INITIAL_NUM_ALLELE_FRACTION_STATES_LONG_NAME,
+            shortName = INITIAL_NUM_ALLELE_FRACTION_STATES_SHORT_NAME,
+            optional = false
+    )
+    protected int initialNumAFStates;
+
+    @Argument(
+            doc = "Input file for allelic-bias panel of normals.",
+            fullName = ExomeStandardArgumentDefinitions.ALLELIC_PON_FILE_LONG_NAME,
+            shortName = ExomeStandardArgumentDefinitions.ALLELIC_PON_FILE_SHORT_NAME,
+            optional = true
+    )
+    protected File allelicPoNFile;
+
+    @Argument(
+            doc = "Output file for copy-ratio segments.  Output is not logged.",
+            fullName = ExomeStandardArgumentDefinitions.SEGMENT_FILE_LONG_NAME,
+            shortName = ExomeStandardArgumentDefinitions.SEGMENT_FILE_SHORT_NAME,
+            optional = false
+    )
+    protected File outputSegmentsFile;
+
+    @Override
+    public Object doWork() {
+        final AllelicPanelOfNormals allelicPoN =
+                allelicPoNFile != null ? AllelicPanelOfNormals.read(allelicPoNFile) : AllelicPanelOfNormals.EMPTY_PON;
+        final AllelicCountCollection acc = new AllelicCountCollection(snpCountsFile);
+        final AlleleFractionSegmenter alleleFractionSegmenter = new AlleleFractionSegmenter(initialNumAFStates, acc, allelicPoN);
+
+        final ReadCountCollection counts;
```

Comment:
Also, would either name this `rcc` as well or change `acc` above to something more verbose.